test/DynamoCoreTests/DynamoModelTestBase.cs

```diff
@@ -36,11 +36,11 @@ public override void Cleanup()
                 preloader = null;
                 DynamoSelection.Instance.ClearSelection();
 
-                if (this.CurrentDynamoModel == null)
-                    return;
-
-                this.CurrentDynamoModel.ShutDown(false);
-                this.CurrentDynamoModel = null;
+                if (this.CurrentDynamoModel != null)
+                {
+                    this.CurrentDynamoModel.ShutDown(false);
+                    this.CurrentDynamoModel = null;
+                }
```

Comment:
In the earlier code, `base.Cleanup()` would have been skipped if `CurrentDynamoModel == null` (but this has nothing to do with the test failure fix).